exonum-java-binding/core/rust/src/proxy/runtime.rs

```diff
@@ -163,6 +163,34 @@ impl Runtime for JavaRuntimeProxy {
         })
     }
 
+    fn initiate_resuming_service(
+        &self,
+        context: ExecutionContext,
+        spec: &InstanceSpec,
+        parameters: Vec<u8>,
+    ) -> Result<(), ExecutionError> {
+        let serialized_instance_spec: Vec<u8> = spec.to_bytes();
+
+        jni_call_default(&self.exec, |env| {
+            let fork_handle = to_handle(View::from_ref_mut_fork(context.fork));
+            let instance_spec =
```

Comment:
We convert InstanceSpec to bytes three times — why dont' we extract this op in a method (possibly, there are other opportunities)?

Reply:
I reused `proto_to_java` function from our utils module - seems better

Reply:
Definitely, thanks!